src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Flyout.cs

```diff
@@ -847,6 +847,44 @@ public async Task When_AttachedFlyout_Popup_XamlRoot()
 			}
 		}
 
+#if __IOS__
+		[TestMethod]
+		[RequiresFullWindow]
+		public async Task When_Native_DatePickerFlyout_Placement()
+		{
+			var flyout = new NativeDatePickerFlyout();
```

Comment:
Is this coming from a missing file? Maybe a namespace change?

Reply:
Didn't realize the type is not available on the other targets, fixed.